Description: Fix mariadb_config --cflags output
 Without this patch, the output starts with something like
 "-I/usr/include/mariadb-g -O2".  This fixes the output to put a space
 between the -I parameter and -g.
Author: Daniel Schepler <schepler@debian.org>

--- mariadb-client-lgpl-2.0.0.orig/mariadb_config/mariadb_config.c.in
+++ mariadb-client-lgpl-2.0.0/mariadb_config/mariadb_config.c.in
@@ -6,7 +6,7 @@
 #define INCLUDE "-I@CMAKE_INSTALL_PREFIX@/include/mariadb"
 #define LIBS    "-L@CMAKE_INSTALL_PREFIX@/@LIB_INSTALL_DIR@ -lmariadb" \
                 "@extra_dynamic_LDFLAGS@"
-#define CFLAGS  INCLUDE "@CMAKE_C_FLAGS@"
+#define CFLAGS  INCLUDE " @CMAKE_C_FLAGS@"
 #define VERSION "@MYSQL_CLIENT_VERSION@"
 #define SOCKET  "@MYSQL_UNIX_ADDR@"
 #define PORT "@MYSQL_PORT@"
